### ISSUE TITLE
Connection test when starting a call

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/spreed#13561]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

We should extend the device checker to also:
- [ ] Allow recording audio and playing it (echo service)
- [ ] Have a quick band-width and connection test to the HPB and or turn?
- [ ] Related https://github.com/nextcloud/spreed/issues/12603

**Comment by nickvergessen** [comment]
cc @fancycode @danxuliu

**Comment by fancycode** [comment]
Yeah, it makes sense to test the bandwidth of the connection that will be used in the call later (e.g. directly with the HPB or through TURN).
